docs/config.yaml

```diff
@@ -1,7 +1,8 @@
 cdf_client:
     project: get-power-grid
     client_id: ca4b8f9c6-7d6e-4d7b-9cf3-6d4a32f8b7e1
-    client_secret: Zy7!nM4cFp9sH3gR6tB8kL0oP7eU6wD5vQ4zN9yF
```

Comment:
This look like a cause for calling an incident, even though it it a mock secret.